subsys/bluetooth/mesh/light_ctl_srv.c

```diff
@@ -454,16 +473,16 @@ static int bt_mesh_light_ctl_srv_start(struct bt_mesh_model *model)
 
 	switch (srv->lightness_srv.ponoff.on_power_up) {
 	case BT_MESH_ON_POWER_UP_OFF:
-		bt_mesh_light_temp_srv_set(&srv->temp_srv, NULL, &temp, NULL);
+		bt_mesh_light_temp_srv_set(&srv->temp_srv, NULL, &temp, &temp_status);
```

Comment:
Woah, good catch

Reply:
It was actually working correctly. `bt_mesh_light_temp_srv_set` was checking `status` pointer inside. But I removed this for clarity. It is also internal function, so it doesn't affect public API.